src/components/EditProfileModal/EditProfileModal.jsx

```diff
@@ -0,0 +1,278 @@
+                    <div className="edit-profile-detail-header">
+                        <h3>Cover photo</h3>
+                        <p 
+                            className="profile-edit-text"
+                            onClick={addNewBackgroundPicFile} 
+                        >
+                            Edit
+                        </p>
+                    </div>
+                    {
+                        selectedBackgroundFileUrl!=="" ?
+                        (
+                            <Image
+                            className="edit-details-background-pic"
+                            cloudName="dddfc84ni"
+                            publicId={selectedBackgroundFileUrl}
+                            />
+                        ): (
+                            <img className="edit-details-background-pic" src={userBackgroundPic} alt="user background pic"></img>
+                        )
+                    }                    
+                </div>
+
+                <div className="edit-profile-detail-header bio-header">
+                    <h3>Bio</h3>
+                </div>
+                <div className="updated-profile-bio-container">
+                    <textarea  
+                        className="updated-bio-content"
+                        placeholder="Enter your bio"
+                        value={updatedBioText}
+                        onChange={(event)=>{
+                            setUpdatedBioText(event.target.value)
+                        }}
+                    >
+                    </textarea>
+                </div>
+
+                <div className="add-to-profile-options-container">
+                    <p>Add to your profile bio</p>
+                    <div 
+                        className="emoji-container"
+                        onClick={()=>setShowEmojiPicker(prevState => !prevState)}
+                    >
+                        <BsEmojiSmile className="create-new-profile-icons"/>
+                        {
+                            showEmojiPicker && 
+                            <div
+                                className="emoji-picker-container"
+                                    onClick={event=>{
+                                        event.stopPropagation()
+                                    }}
+                            >
+                                <Picker 
+                                    className="emoji-picker" 
+                                    onEmojiClick={onEmojiClick} 
+                                    skinTone={SKIN_TONE_NEUTRAL} 
+                                />
+                            </div>
+                        }
+                    </div>
+                </div>
+
+                <div className="edit-user-profilepic-container">
+                    <div className="edit-profile-detail-header">
+                        <h3>Portfolio Link</h3>
```

Comment:
would be better if this was a more general term. like "Add a Link" instead of portfolio link. 
Also, the default images can be more neutral grey.
![image](https://user-images.githubusercontent.com/67316487/166416846-12180cb2-2f18-4029-996c-8db108e17c71.png)